### PR TITLE
Feature/issue 566 export alternate name to algolia

### PR DESCRIPTION
**Pull request by ian-lin-PG**

### CODE REVIEW

File: utility/algolia/dump_services.sql [hunk 1]
```diff
@@ -18,6 +18,7 @@ select distinct
 	'"' || serviceStrings.name || '"' as service_name,
 	left(regexp_replace(serviceStrings.description, E'[\\n\\r\\t;,]+', ' ', 'g' ), 5000) as service_description,
     service.last_verified_date as last_verified_date,
+	serviceStrings.alternate_name as alternate_name,
```
Comment (rasmus-storjohann-PG) [inline utility/algolia/dump_services.sql:21]: Alternate names can presumably contain whitespace much like name, so it needs to be quoted in the same way, see patterns to follow above and  below.

File: utility/algolia/dump_services.sql [hunk 2]
```diff
@@ -3,7 +3,7 @@
 -- sudo npm install --global csvtojson
 
 -- psql -d pathways_local -F $',' -A -t -c 'select location_id, phone_number, phone_number_type_id from phone_at_location_phoneatlocation' > phones_data.csv
--- psql -d pathways_local -F $',' -A -t -c "select s.service_id, t.name from services_service_taxonomy_terms as s, taxonomies_taxonomyterm as t where s.taxonomyterm_id=t.id and t.taxonomy_id like 'bc211-%' " > taxonomy_data.csv
+-- psql -d pathways_local -F $',' -A -t -c "select s.service_id, t.name from services_service_taxonomy_terms as s, taxonomies_taxonomyterm as t where s.taxonomyterm_id=t.id" > taxonomy_data.csv
```
Comment (rasmus-storjohann-PG) [inline utility/algolia/dump_services.sql:6]: This change should not be needed, working on fixing/confirming that this will work as before

Comment (rasmus-storjohann-PG) [reply utility/algolia/dump_services.sql:6]: With my PRs merged the other day, this change should no longer be needed, @ian-lin-PG

Comment (ian-lin-PG) [reply utility/algolia/dump_services.sql:6]: Confirmed with staging pass 1.58.0. closing this issue.

**Pull request closed without merging**